### ISSUE TITLE
Support ES256

### ISSUE THREAD

**Issue by george-angel** [mike-engel/jwt-cli#12]
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Json(Error("unknown variant `ES256`, expected one of `HS256`, `HS384`, `HS512`, `RS256`, `RS384`, `RS512`", line: 1, column: 14)))', src/libcore/result.rs:997:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:70
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:58
             at src/libstd/panicking.rs:200
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:215
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:478
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:385
   6: rust_begin_unwind
             at src/libstd/panicking.rs:312
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::result::unwrap_failed
   9: jwt::main
  10: std::rt::lang_start::{{closure}}
  11: std::panicking::try::do_call
             at src/libstd/rt.rs:49
             at src/libstd/panicking.rs:297
  12: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:87
  13: std::rt::lang_start_internal
             at src/libstd/panicking.rs:276
             at src/libstd/panic.rs:388
             at src/libstd/rt.rs:48
  14: main
  15: __libc_start_main
  16: <unknown>
```

I'm not very well versed in the realm of tokens, but I'm guessing missing support for ECDSA? Would this be difficult to add?

Thanks!

**Comment by mike-engel** [comment]
Hey @george-angel, thanks for the issue! The version of [`jsonwebtoken`](https://github.com/Keats/jsonwebtoken) that was out when I made the last version didn't support ES256/384. It looks like there's a new version that does, so I'll need to update to the latest version of that library. Not sure when I'll be able to do that, maybe this week :)

**Comment by george-angel** [comment]
No problem, its ready when its ready ))

Thanks!

**Comment by mike-engel** [comment]
@george-angel Thanks again for this issue—it forced me to update that dependency, which is always good! 

I've released 2.5.0 which has support now for ES256 and ES384. Keep in mind that for ECDSA specifically, only a single curve is supported. You can read more here: https://gist.github.com/briansmith/2ee42439923d8e65a266994d0f70180b#generating-private-keys

If you use homebrew, you should be able to run `brew upgrade mike-engel/jwt-cli/jwt-cli`. If you use another platform, those binaries will be available shortly :)

**Comment by george-angel** [comment]
Luckily my pki is using P-256, so my use case is covered. Got `2.5.0` - can verify it is working fine.

Thanks Mike! Much appreciated.
